db/migrate/20201104003549_create_drivers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDrivers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :drivers do |t|
+      t.string :name
+      t.string :vin
+      t.string :available
```

Comment:
Since you only have two legal values "available" should have been a boolean, not a string:

```suggestion
      t.boolean :available
```